### DISCUSSION TITLE
Twilio

### DISCUSSION THREAD

**Discussion by ATLCOTech** [code4recovery/12-step-meeting-list#1386] [General]
Hi is anyone familiar with Twilio? The previous web/tech committee used it and I have been unable to get them to return my calls and cannot figure out how to use this platform. Can anyone help the Atlanta Central Office in using this platform?

**Comment by anchovie91471** [comment]
My district uses it for notifications when people sign up to speak at a
correctional facility or treatment center. It's very confusing and they are
terrible about answering tickets, etc., but I do have some experience with
it.

Anthony B.

On Tue, Mar 12, 2024 at 9:21 AM Mark Just ***@***.***> wrote:

> Hi is anyone familiar with Twilio? The previous web/tech committee used it
> and I have been unable to get them to return my calls and cannot figure out
> how to use this platform. Can anyone help the Atlanta Central Office in
> using this platform?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/1386>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACF42JJA24ZJ3I53C3RYLOLYX4FO7AVCNFSM6AAAAABESK3NKWVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGM3DGMRVGY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
*Anthony Baggett* || President // Founder
104 N Broadway St.

#2

Tupelo, MS 38804


c. <http://goog_2089110092>662.610.0728 <6626100728>


*BustedSpring.com <http://bustedspring.com/>*- - - - - - - - - - - - - - -
- - - - - - - - - - - - - - - - -

**Reply by ATLCOTech** [reply]
Do you have some time this week to chat?

**Comment by anchovie91471** [comment]
Sure. My Wednesdays are always crazy, but Thursday and Friday I can do it.

On Tue, Mar 12, 2024 at 9:34 AM Mark Just ***@***.***> wrote:

> Do you have some time this week to chat?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/1386#discussioncomment-8760815>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACF42JOIQE2TZPKEPOKCAHLYX4G6BAVCNFSM6AAAAABESK3NKWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DONRQHAYTK>
> .
> You are receiving this because you commented.Message ID:
> <code4recovery/12-step-meeting-list/repo-discussions/1386/comments/8760815
> @github.com>
>


-- 

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
*Anthony Baggett* || President // Founder
104 N Broadway St.

#2

Tupelo, MS 38804


c. <http://goog_2089110092>662.610.0728 <6626100728>


*BustedSpring.com <http://bustedspring.com/>*- - - - - - - - - - - - - - -
- - - - - - - - - - - - - - - - -
